wormhole-connect/src/components/TokensModal.tsx

```diff
@@ -511,6 +511,16 @@ function TokensModal(props: Props) {
       const isNonzeroBalance = b !== null && b !== '0';
       return isNonzeroBalance;
     });
+
+    // If a destination token is native, display only that token.
+    // This is to avoid users inadvertently receiving wrapped versions of the token.
+    if (type === 'dest') {
+      const nativeToken = supportedTokens.find((t) => t.nativeChain === chain);
+      if (nativeToken) {
+        filtered = [nativeToken];
```

Comment:
is it possible there's multiple native tokens eventually? like if Connect can perform swaps?

this logic is basically "find the first native token and if it exists, only offer that." we could also have it be "find all the native tokens, if any exist, offer them all."

Reply:
I think we can change it to be "future proof"

Reply:
changed